content/en/platform/corda/1.4/cenm/config-database.md

```diff
@@ -15,41 +15,33 @@ title: CENM Database Configuration
 
 
 # CENM Database Configuration
-<!-- Sentence below: needs clarity - to be available needs to be rewritten to make it more meaningful.. -->
-Both the Network Map and Identity Manager Components require a persistent layer to be available. This is described in
-their respective configs with the following configuration state.
-
+A persistent layer must be available for the Identity Manager and Network Map Components to function. This is described in
+their respective configs with the following configuration states.
 
 * **database**:
+Database configuration.
 
 * **runMigration**:
 Create or upgrade the database schema (database objects like tables, indices) to the current version of the service. If it is set to false, and the database schema is up to date, the service will validate.
 
 * **initialiseSchema**:
 *(Deprecated)* Automatically creates the tables required by the CENM component.
-{{< note>}} This property was used for H2 database only. It has  been ignored since CENM 1.2 and has been replaced by `runMigration` {{< /note>}}
-
+{{< note>}} This property was used for H2 database only. It has  been ignored since CENM 1.2 and has been replaced by `runMigration`. {{< /note>}}
```

Comment:
```suggestion
{{< note>}} This property was used for H2 database only. It has been ignored since CENM 1.2 and has been replaced by `runMigration`. {{< /note>}}
```